### PR TITLE
Judgment details links to search

### PR DESCRIPTION
**Pull request by pawelczak**
#728

### CODE REVIEW

File: saos-webapp/src/main/resources/message/common.properties [hunk 1]
```diff
@@ -296,6 +296,21 @@ judgmentDetails.corrections.value.DECISION = Postanowienie
 judgmentDetails.corrections.value.REGULATION = Zarządzenie
 judgmentDetails.corrections.value.OTHER = Inny
 
+
+judgmentDetails.linkTooltip.judgmentType = Pokaż orzeczenia z takim typem orzeczenia
```
Comment (madryk) [inline saos-webapp/src/main/resources/message/common.properties]: what do you think about: `Pokaż orzeczenia o takim typie` ?

File: saos-webapp/src/main/resources/message/common.properties [hunk 2]
```diff
@@ -296,6 +296,21 @@ judgmentDetails.corrections.value.DECISION = Postanowienie
...
+judgmentDetails.linkTooltip.date = Pokaż orzeczenia o dacie
+judgmentDetails.linkTooltip.courtType = Pokaż orzeczenia z takim organem orzekającym
+judgmentDetails.linkTooltip.commonCourt = Pokaż orzeczenia z sądu
+judgmentDetails.linkTooltip.commonCourtDivision = Pokaż orzeczenia z tego sądu i wydziału
+judgmentDetails.linkTooltip.scCourtChamber = Pokaż orzeczenia z taką izbą
+judgmentDetails.linkTooltip.scCourtChamberDivision = Pokaż orzeczenia dla jednostki obsługującej sprawę
+judgmentDetails.linkTooltip.scJudgmentForm = Pokaż orzeczenia z taką formą orzeczenia
+judgmentDetails.linkTooltip.scPersonnelType = Pokaż orzeczenia z takim typem składu sędziowskiego
```
Comment (madryk) [inline saos-webapp/src/main/resources/message/common.properties]: `Pokaż orzeczenia z takim składem sędziowskim`?

File: saos-webapp/src/main/webapp/WEB-INF/view/details/judgmentDetails.jsp [hunk 3]
```diff
@@ -259,12 +294,16 @@ $(document).ready(function() {
...
 			
 			<c:if test="${!empty judgment.keywords}" >
+			    <spring:message code="judgmentDetails.linkTooltip.keywords" var="keywordsLinkTooltip" />
 				<li>
 					<div class="" >
 						<div class="label-title" ><spring:message code="judgment.keywords" />:</div>
 						<div class="desc" >
 							<c:forEach items="${judgment.keywords}" var="keyword" varStatus="status">
-								<c:out value="${keyword.phrase}" /><c:if test="${!status.last}">,</c:if>
+							<a href="${contextPath}/search?courtCriteria.courtType=COMMON&amp;keywords=${keyword.phrase}" data-toggle="tooltip" title="${keywordsLinkTooltip}" >
```
Comment (madryk) [inline saos-webapp/src/main/webapp/WEB-INF/view/details/judgmentDetails.jsp]: If we would have keywords not only in common court judgments, then `?courtCriteria.courtType=${judgment.courtType}` would be more generic

File: saos-webapp/src/main/resources/message/common.properties (hunk 1 above)
Comment (pawelczak) [inline saos-webapp/src/main/resources/message/common.properties]: :+1:

**Pull request closed without merging**